boards/cats_rev1Pro/Core/Src/tasks/task_imu_read.c

```diff
@@ -57,20 +58,29 @@ void task_imu_read(void *argument) {
     //            2, gyroscope[0], gyroscope[1], gyroscope[2],
     //            acceleration[0], acceleration[1], acceleration[2], temperature);
 
+    /* Read and Save Magnetometer Data */
     mmc5983ma_read_calibrated(&MAG, data);
     magneto_data.magneto_x = data[0];
     magneto_data.magneto_y = data[1];
     magneto_data.magneto_z = data[2];
     magneto_data.ts = osKernelGetTickCount();
-    log_info("Magneto %d: RAW Mx: %ld, My:%ld, Mz:%ld", 1, (int32_t)((float)data[0] * 1000),
-             (int32_t)((float)data[1] * 1000), (int32_t)((float)data[2] * 1000));
+    // log_info("Magneto %ld: RAW Mx: %ld, My:%ld, Mz:%ld", 1, (int32_t)((float)data[0] * 1000),
+    //         (int32_t)((float)data[1] * 1000), (int32_t)((float)data[2] * 1000));
 
     global_magneto = magneto_data;
     record(MAGNETO, &magneto_data);
 
+    /* Read and Save high G IMU Data */
+    /* Todo: Read high G IMU Data */
+    record(ACCELEROMETER, &accel_data);
+    global_imu[2].ts = (accel_data.ts);
+    global_imu[2].acc_x = (int16_t)(accel_data.acc_x);
+    global_imu[2].acc_y = (int16_t)(accel_data.acc_y);
+    global_imu[2].acc_z = (int16_t)(accel_data.acc_z);
+
     /* TODO: The speed of copying looks to be the same, code size reduced by 16B
      * with memcpy vs. assignment with -0g */
-    for (int i = 0; i < 2; i++) {
+    for (int i = 0; i < 1; i++) {
```

Comment:
This is an example location where Luca's suggestion for a configurable sensor setup shows benefits.